### PR TITLE
Eco_cosmetics

### PR DESCRIPTION
**Pull request by Andrii-Monko**
[DEMO LINK](https://Andrii-Monko.github.io/Eco_cosmetics/)

### CODE REVIEW

**Review by etojeDenys — CHANGES_REQUESTED** [review]
1. let's center container on wide screens
image


2. you should stretch these lines to the edges of the screen
image


3. make all form fields required 
4. add some hovers to links
image

File: src/index.html [hunk 1]
```diff
@@ -1,13 +1,537 @@
...
+            alt="instagram-icon"
+            class="footer__icon"
+          >
+        </a>
+      </div>
+
+
+      <div class="contact__information">
+        <p class="contact__text">
+          Ukrain
```
Comment (volodymyr-soltys97) [inline src/index.html:511]:
```suggestion
          Ukraine
```

File: src/index.html [hunk 2]
```diff
@@ -1,13 +1,537 @@
...
+        </p>
+
+        <p class="contact__box">
+          <a href="mail:support.mountain@gmail.com" class="contact__text">
```
Comment (volodymyr-soltys97) [inline src/index.html:515]:
```suggestion
          <a href="mailto:support.mountain@gmail.com" class="contact__text">
```

File: src/index.html [hunk 3]
```diff
@@ -1,13 +1,537 @@
...
+            alt="death_bird_photo"
+            class="tooth__photo-first"
+          >
+          
+          <p class="tooth__photo-describe">
+            product packaging can be recycled or reused
+          </p>
+        </div>
+
+        <div class="p">
```
Comment (volodymyr-soltys97) [inline src/index.html:380]: Use a more descriptive classes name instead of p

File: src/index.html [hunk 4]
```diff
@@ -1,13 +1,537 @@
...
+      </div>
+    </section>
+    <!-- #endregion  -->
+
+    <!-- #region GRAPHIC-PHOTO -->
+    <section class="tooth">
+      <div class="tooth__left-size">
+        <div class="left-size-text">
+          Ingredients
+        </div>
```
Comment (volodymyr-soltys97) [inline src/index.html:313]: Use a semantic tags everywhere for text

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
1. The background color should stretch to the full width as it is on the design
image

2. Add hover effects for all clickable elements
image

3. Make all these fields are required before sending
image

**Review by loralevitska — CHANGES_REQUESTED** [review]
You did an amazing job! Looks good! Let's do some improvements (check comments)

- add transition for underlining
- don't reload page after form submit, just send form and clear after that
- fix image jumping due to underline
image
- 2023 
image
- make smooth menu appear/disappear 
- fix it according design (tablet mode)
image
- the cross icon in menu should be on the same position with burger-menu icon on page, please fix it

**Review by IvanFesenko — CHANGES_REQUESTED** [review]
- Hover effects should not effect other elements
https://github.com/mate-academy/Eco_cosmetics/assets/61904995/5aae6406-74f5-4343-a9bd-d51fee72de7d
- Look like redundant margin
2023-10-31_13h31_24
- disable scroll in case menu is open
 - Add cursor: pointer to elements that are clickable- product card for example

**Review by Andriy-Shymkiv — APPROVED** [review]

**Pull request closed without merging**